docs/docs/training-data-format.mdx

```diff
@@ -428,6 +428,42 @@ policies learn to predict the next action based on a *combination* of
 both the intent and entities (you can, however, change this behavior
 using the [`use_entities`](#entities) attribute).
 
+If you use `user:` key while writing stories, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline. Instead, you can simply put
+the text of the user message.
+We call it end-to-end training because actual text is used for predictions.
```

Comment:
If we call it that, then we should also add it to the glossary (are we in sync with marketing / CSE about the naming?)

Reply:
I don't think we have a name. It's rather name widely used in the field